### PR TITLE
[Bug]: fix *.launch *.refresh not found bug

### LINKED ISSUE #277: Error: "command 'vscode-edge-devtools-view.launch' not found
Seeing this error pop up occasionally when trying to create a new target.  In my case, this is a transient issue where trying to launch again generally fixes it.

### PR DESCRIPTION
**Pull request by mliao95**
Issue:
- This message pops up when users press either the `Open a new tab` or `refresh` buttons at the top before the commands can be registered
image

Changes:
- added `titleCommandsRegistered` condition to show the `Open a new tab` or `refresh` buttons
    - the `titleCommandsRegistered` flag is turned to true once registration for those commands are complete.

closes #277

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -1,234 +1,235 @@
...
-            const match = exeName!.match(/(chrome|edge)/gi) || [];
-            const knownBrowser = match.length > 0 ? match[0] : "other";
-            const browserProps = { exe: `${knownBrowser.toLowerCase()}` };
-            telemetryReporter.sendTelemetryEvent("command/launch/browser", browserProps);
-        }
-        browserInstance = await launchBrowser(browserPath, port, url, userDataDir);
-        await attach(context, url, config);
-    }
-}
+// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (vidorteg) [inline src/extension.ts:1]: not sure why but the file shows all the content as new/changed which makes reviewing it really difficult. Could you double check and scope the review to the changes?

**Review by vidorteg — CHANGES_REQUESTED** [review]
Left some comments

File: src/extension.ts (hunk 1 above)
Comment (mliao95) [reply src/extension.ts:1]: It looks like this is happening whenever we use Github's rebase... Not sure why, but I'll try fixing this.

Comment (mliao95) [reply src/extension.ts]: Okay fixed and ready for review

**Review by vidorteg — APPROVED** [review]
This seems like a difficult issue to spot, thanks for fixing it @mliao95

**Pull request merged**